examples/01-blog/Blog/Type/CommentType.php

```diff
@@ -19,9 +18,27 @@ public function __construct()
             'name' => 'Comment',
             'fields' => static fn (): array => [
                 'id' => Types::id(),
-                'author' => Types::user(),
-                'parent' => Types::comment(),
-                'isAnonymous' => Types::boolean(),
+                'author' => [
+                    'type' => Types::user(),
+                    'resolve' => function (Comment $comment): ?User {
+                        if ($comment->isAnonymous) {
+                            return null;
+                        }
+
+                        return DataSource::findUser($comment->authorId);
+                    },
+                ],
+                'parent' => [
+                    'type' => Types::comment(),
+                    'resolve' => function (Comment $comment): ?Comment {
+                        if ($comment->parentId !== null) {
+                            return DataSource::findComment($comment->parentId);
+                        }
+
+                        return null;
+                    },
+                ],
+                'isAnonymous' => ['type' => Types::boolean(), 'resolve' => static fn (Comment $comment): bool => $comment->isAnonymous],
```

Comment:
Can we just omit `resolve` here? The default resolver should do this.

Reply:
Yes! I forgot that was a thing, thank you 6060c965